src/XrdTpc/XrdTpcTPC.cc

```diff
@@ -849,6 +869,8 @@ int TPCHandler::ProcessPushReq(const std::string & resource, XrdHttpExtReq &req)
 //  curl_easy_setopt(curl, CURLOPT_SOCKOPTFUNCTION, sockopt_setcloexec_callback);
     curl_easy_setopt(curl, CURLOPT_OPENSOCKETFUNCTION, opensocket_callback);
     curl_easy_setopt(curl, CURLOPT_OPENSOCKETDATA, &rec);
+    curl_easy_setopt(curl, CURLOPT_CLOSESOCKETFUNCTION, closesocket_callback);
+    curl_easy_setopt(curl,CURLOPT_CLOSESOCKETDATA,&rec);
```

Comment:
Just noticed the missing space after the comma, different than the ones above it. Not a critical thing, though.

Reply:
Yep, now it's cleaner :+1: